setup.py

```diff
@@ -4,7 +4,7 @@
 from setuptools import setup
 
 version = '1.0.dev0'
-description = 'Portlets para Portal PadrÄĹo'
+description = 'Portlets para Portal Padrao'
```

Comment:
this is been caused by zest.releaser but I have no idea why and how to fix it; we better ask @reinout and @mauritsvanrees.